examples/kfp-pipelines/superworkflows/ray/kfp_v2/README.md

```diff
@@ -28,6 +30,9 @@ Another useful feature of the KFP v2 is the `Json` editor for the `dict` type in
 cd examples/kfp/superworkflows/ray/kfp_v2/
 make clean
 export KFPv2=1
-export PYTHONPATH=../../../../../transforms
 make workflow-build
 ```
+
+Please be aware that in KFP v2, there is an (issue)[default pipeline input params are missing from the GUI](https://github.com/kubeflow/pipelines/issues/11515]] where default values for pipeline input parameters do not appear in the GUI, 
```

Comment:
I think it is relevant only for booleans with default `False` and empty strings.
BTW we can easily overcome the boolean case, just use the inverted value ;-)